trunk/SUAVE/Components/Energy/Converters/Propeller.py

```diff
@@ -58,10 +59,15 @@ def __defaults__(self):
         self.prop_attributes.twist_distribution  = 0.0
         self.prop_attributes.chord_distribution  = 0.0
         self.prop_attributes.mid_chord_alignment = 0.0
+        self.prop_attributes.lift_curve_slope    = 2.*np.pi    
+        self.prop_attributes.cd_coeff            = [.108, -.2612, .181, -.0139, .0278]  #coefficients for a 4th degree polynomial fit of Cd(Cl) for the airfoil
+        self.prop_attributes.cd_re_scaling_coeff = [50000., .2]  # scaling = (coeff[0]/Re) **coeff[1]
```

Comment:
I think we should split out the reynolds scaling. The first number is the reference value at which the drag polar was obtained. The second number is an empirical coefficient from the 241 notes for the reynolds effects of drag.

Also maybe we should change the names. Perhaps:

cd_coefficients
drag_reference_reynolds_number
reynolds_scaling_exponent

Thoughts?

Reply:
Works for me. Done